mixins/form.jsx

```diff
@@ -264,13 +265,13 @@ module.exports = {
       exp_year: submitProps.expYear
     }, function(status, response) {
       if (response.error) {
-        error(response.error);
+        error(response.error.code, response.error.type);
```

Comment:
This is the main fix. createToken errors return error.type, and stripe.charges.create return error.rawType.

Both are the values we want.